README.md

```diff
@@ -69,9 +69,23 @@ Or you can submit all in one transaction.
 
 `Oracle` is a wrapper object of `DNSSEC.sol` Oracle smart contract.
 
-- `knownProof(proof)` returns true if the given proof already exists in `Oracle`.
-- `submit(dnsresult, params)` sends all unproven proofs into DNSSEC Oracle as one transaction in a batch.
+- `submitAll(dnsresult, params)` sends all unproven proofs into DNSSEC Oracle as one transaction in a batch.
+- `getAllProofs(dnsresult)` returns all the proofs needs to be submitted into DNSSEC Oracle. It travarses from the leaf of the chain of proof to check if proof in DNSSEC Oracle and the one from DNS record matches with valid inception value. This function is used so that it can pass the necessary proof to `dnsregistrar.proveAndClaim` function.
```

Comment:
```suggestion
- `getAllProofs(dnsresult)` returns all the proofs needs to be submitted into DNSSEC Oracle. It traverses from the leaf of the chain of proof to check if proof in DNSSEC Oracle and the one from DNS record matches with valid inception value. This function is used so that it can pass the necessary proof to `dnsregistrar.proveAndClaim` function.
```